spec/bracket-matcher-spec.coffee

```diff
@@ -242,6 +242,26 @@ describe "bracket matching", ->
           editor.setCursorBufferPosition([2, Infinity])
           expectHighlights([2, 14], [2, 3])
 
+        it "highlights the correct opening tag, skipping self-closing tags", ->
+          buffer.setText """
+            <test>
+              <test />
+            </test>
+          """
+
+          editor.setCursorBufferPosition([2, Infinity])
+          expectHighlights([2, 2], [0, 1])
+
+      describe "when on a self-closing tag", ->
```

Comment:
I would like two extra tests:
1. Test to make sure `<test/>` is identified as a self-closing tag
2. Test to make sure `<test______/>` (where `_` is a space) is identified as a self-closing tag